src/AppBundle/Validator/ValidAddressValidator.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace AppBundle\Validator;
+
+use AppBundle\Address\AddressInterface;
+use AppBundle\Intl\FranceCitiesBundle;
+use Symfony\Component\PropertyAccess\PropertyAccessor;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+/**
+ * @Annotation
+ */
+class ValidAddressValidator extends ConstraintValidator
```

Comment:
The alias in the use statement does not have to be named as the constraint class. Naming the constraint this way implies you only validate valid address, using `AssertValidAddress` as alias for the annotation makes more sense.